app/models/adoption_application.rb

```diff
@@ -0,0 +1,18 @@
+class AdoptionApplication < ApplicationRecord
+  validates_presence_of :name, :address, :city, :state, :zip, :phone_number, :description
+  has_many :pet_applications
+  has_many :pets, through: :pet_applications
+
+  def self.pets
```

Comment:
This is the first class method that I've written in rails. Feeling pretty good about the functionality but not that stoked on the nested iteration. I think I can probably do it in less lines, but I think we've got bigger fish to fry.

Reply:
We can always return to this if we feel it necessary.